modules/features/sharing/src/main/kotlin/au/com/shiftyjelly/pocketcasts/sharing/ui/VerticalCard.kt

```diff
@@ -0,0 +1,220 @@
+internal fun VerticalEpisodeCard(
+    episode: PodcastEpisode,
+    podcast: Podcast,
+    useEpisodeArtwork: Boolean,
+    shareColors: ShareColors,
+    modifier: Modifier = Modifier,
+) = VerticalCard(
+    type = EpisodeCardType(
+        episode = episode,
+        podcast = podcast,
+        useEpisodeArtwork = useEpisodeArtwork,
+    ),
+    shareColors = shareColors,
+    modifier = modifier,
+)
+
+@Composable
+private fun VerticalCard(
+    type: CardType,
+    shareColors: ShareColors,
+    modifier: Modifier = Modifier,
+) = BoxWithConstraints {
+    val backgroundGradient = Brush.verticalGradient(
+        listOf(
+            shareColors.cardTop,
+            shareColors.cardBottom,
+        ),
+    )
+    Column(
+        horizontalAlignment = Alignment.CenterHorizontally,
+        modifier = modifier
+            .background(backgroundGradient, RoundedCornerShape(12.dp))
+            .width(maxWidth)
+            .height(maxWidth * 1.5f),
+    ) {
+        Spacer(
+            modifier = Modifier.height(this@BoxWithConstraints.maxWidth * 0.15f),
+        )
+        type.Image(
+            modifier = Modifier
+                .size(this@BoxWithConstraints.maxWidth * 0.7f)
+                .clip(RoundedCornerShape(8.dp)),
+        )
+        Spacer(
+            modifier = Modifier.height(24.dp),
+        )
+        TextH70(
+            text = type.topText(),
+            maxLines = 1,
+            color = shareColors.cardText.copy(alpha = 0.5f),
+            modifier = Modifier.padding(horizontal = 64.dp),
+        )
+        Spacer(
+            modifier = Modifier.height(6.dp),
+        )
+        TextH40(
+            text = type.middleText(),
+            maxLines = 2,
+            textAlign = TextAlign.Center,
+            color = shareColors.cardText,
+            modifier = Modifier.padding(horizontal = 42.dp),
+        )
+        Spacer(
+            modifier = Modifier.height(6.dp),
+        )
+        TextH70(
+            text = type.bottomText(),
+            maxLines = 2,
+            textAlign = TextAlign.Center,
+            color = shareColors.cardText.copy(alpha = 0.5f),
+            modifier = Modifier.padding(horizontal = 64.dp),
+        )
+        Spacer(
+            modifier = Modifier.weight(1f),
+        )
+        Row(
+            verticalAlignment = Alignment.CenterVertically,
+            modifier = Modifier
+                .background(Color(0xFFF43E37), RoundedCornerShape(24.dp))
+                .height(24.dp)
+                .padding(start = 4.dp, end = 8.dp),
+        ) {
+            Image(
+                painter = painterResource(id = R.drawable.ic_logo_foreground),
+                contentDescription = null,
+                modifier = Modifier.size(18.dp),
+            )
+            Spacer(
+                modifier = Modifier.width(8.dp),
+            )
+            TextH70(
+                text = "Pocket Casts",
+                color = Color.White,
+            )
+        }
```

Comment:
Can we make this pill a reusable component?

Reply:
Sure, I'll extract it.

Reply:
Change in https://github.com/Automattic/pocket-casts-android/pull/2518/commits/a08263f10250915650219101d6452562c67cb033